Snake/SnakeTest/GameSnake_Test.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Linq;
+using System.Text;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace SnakeTest
+{
+    [TestClass]
+    public class GameSnake_Test
+    {
+        [TestMethod]
+
+        public void First_Snake_Created_at_Correct_Location()
+        {
+            //create new snake 
+            BusinessLayer.GameSnake testSnake = new BusinessLayer.GameSnake();
+            List<BusinessLayer.Point> testSnakeList;
+            testSnakeList = testSnake.createFirstSnake(40, 40, 3);

```

Comment:
consider if this magic coordinate should be either a constant or  public readonly of the GameSnake